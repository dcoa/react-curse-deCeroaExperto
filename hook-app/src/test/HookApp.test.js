import '@testing-library/jest-dom'
import { shallow } from 'enzyme';
import { HookApp } from '../HookApp';



describe('Test on component <HookApp />', () => {
    
    test('should shown <HookApp/>', () => {
        const wrapper = shallow(<HookApp/>);

        expect(wrapper).toMatchSnapshot();

    })
    
})