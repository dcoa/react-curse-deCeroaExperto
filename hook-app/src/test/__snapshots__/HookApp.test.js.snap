// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test on component <HookApp /> should shown <HookApp/> 1`] = `
<div>
  Hola mundo
</div>
`;
